tests/background_cpu_test.py

```diff
@@ -67,85 +81,64 @@ def _CheckVMFromSpec(self, spec, num_working):
       self.assertEqual(vm0.RemoteCommand.call_count,
                        expected_remote_post_stop)
 
-
   def testWindowsVMCausesError(self):
     """ windows vm with background_cpu_threads raises exception """
-    with mock_flags.PatchFlags() as mocked_flags:
-      self.setupCommonFlags(mocked_flags)
-      mocked_flags['background_cpu_threads'].Parse(1)
-      mocked_flags['os_type'].Parse(os_types.WINDOWS)
-      config = configs.LoadConfig(ping_benchmark.BENCHMARK_CONFIG, {}, NAME)
-      spec = benchmark_spec.BenchmarkSpec(config, NAME, UID)
-      spec.ConstructVirtualMachines()
-      with self.assertRaisesRegexp(Exception, 'NotImplementedError'):
-        spec.Prepare()
-      with self.assertRaisesRegexp(Exception, 'NotImplementedError'):
-        spec.StartBackgroundWorkload()
-      with self.assertRaisesRegexp(Exception, 'NotImplementedError'):
-        spec.StopBackgroundWorkload()
+    self._mocked_flags['background_cpu_threads'].Parse(1)
```

Comment:
What does this line do? I don't see why this won't just call Parse() on a mock.MagicMock, but it must do more than that.